lasy/utils/laser_utils.py

```diff
@@ -2,10 +2,15 @@
 from scipy.constants import c, epsilon_0, e, m_e
 from scipy.interpolate import interp1d
 from scipy.signal import hilbert
-from skimage.restoration import unwrap_phase
+
+try:
```

Comment:
You can move this import closer to the usage, which will make imports of lasy faster (if scikit-image is present but not used)